documentation/source/overview/references.rst

```diff
@@ -47,69 +47,66 @@ Main reference
-   estimation of partial volume effect. Neuroimage
-   2015 <https://www.ncbi.nlm.nih.gov/pubmed/26099457>`__
--  `De Leener B, Fonov VS, Louis Collins D, Callot V, Stikov N,
-   Cohen-Adad J. PAM50: Unbiased multimodal template of the brainstem
-   and spinal cord aligned with the ICBM152 space. Neuroimage
-   2017. <http://www.sciencedirect.com/science/article/pii/S1053811917308686>`__
+The table below provides individual references for novel methods used in SCT's :ref:`command-line-tools`.
+
+.. note::
+   If you are using segmentation tools (``sct_propseg``/``sct_deepseg``) and registration tools (``sct_register_to_template``/``sct_register_multimodal``) together as part of a pipeline, please also consider this reference:
+
+   `Dupont SM, De Leener B, Taso M, Le Troter A, Stikov N, Callot V, Cohen-Adad J. Fully-integrated framework for the segmentation and registration of the spinal cord white and gray matter. Neuroimage 2017 <https://www.ncbi.nlm.nih.gov/pubmed/27663988>`__
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
 
-Segmentation
-^^^^^^^^^^^^
+   * - Command line script
+     - References
+   * - ``sct_deepseg_gm``
+     - * `Perone et al. Spinal cord gray matter segmentation using deep dilated convolutions. Sci Rep 2018 <https://www.nature.com/articles/s41598-018-24304-3>`__
+   * - ``sct_deepseg_lesion``
+     - * `Gros et al. Automatic segmentation of the spinal cord and intramedullary multiple sclerosis lesions with convolutional neural networks. Neuroimage 2019 <https://www.sciencedirect.com/science/article/pii/S1053811918319578>`__
+   * - ``sct_deepseg_sc``
+     - * `Gros et al. Automatic segmentation of the spinal cord and intramedullary multiple sclerosis lesions with convolutional neural networks. Neuroimage 2019 <https://www.sciencedirect.com/science/article/pii/S1053811918319578>`__
+   * - ``sct_get_centerline``
+     - * `Gros et al. Automatic spinal cord localization, robust to MRI contrasts using global curve optimization. Med Image Anal 2018 <https://www.sciencedirect.com/science/article/pii/S136184151730186X>`__
+   * - ``sct_label_vertebrae``
+     - * `Ullmann et al. Automatic labeling of vertebral levels using a robust template-based approach. Int J Biomed Imaging 2014 <http://downloads.hindawi.com/journals/ijbi/2014/719520.pdf>`__
+   * - ``sct_propseg``
+     - * `De Leener et al. Robust, accurate and fast automatic segmentation of the spinal cord. Neuroimage 2014 <https://www.ncbi.nlm.nih.gov/pubmed/24780696>`__
+       * `De Leener et al. Automatic segmentation of the spinal cord and spinal canal coupled with vertebral labeling. IEEE Transactions on Medical Imaging 2015 <https://www.ncbi.nlm.nih.gov/pubmed/26011879>`__
+   * - ``sct_register_multimodal``
+     - * `Cohen-Adad et al. Slice-by-slice regularized registration for spinal cord MRI: SliceReg. Proc ISMRM 2015 <https://www.dropbox.com/s/v3bb3etbq4gb1l1/cohenadad_ismrm15_slicereg.pdf?dl=0>`__
```

Comment:
I would create a new row for this citation, and associate it with: `sct_register_multimodal -param algo=slicereg`